src/com/nutrons/nu17/RobotMap.java

```diff
@@ -2,41 +2,22 @@
 
 public class RobotMap {
 
-	// Ports of wheels
 	public static final int FRONT_LEFT = 0;
 	public static final int FRONT_RIGHT = 1;
 	public static final int BACK_LEFT = 2;
 	public static final int BACK_RIGHT = 3;
-	
-	// Ports of intake
 	public static final int ROLLER_A = 0;
```

Comment:
Which ones are A and B?  Top/Bottom? Left/Right?  Specify.  If you don't know, put that in a TODO comment